94.md

```diff
@@ -0,0 +1,49 @@
+NIP-94 - File Header
+======
+`draft` `optional` `author:frbitten` 
+
+The purpose of this NIP is to allow an organization and classification of shared files. So that relays can filter and organize in any way that is of interest.
+Also the goal is to create a base on the protocol for this bountsr "Filesharing App" (https://bountsr.org/p2p-filesharing/) to be implemented.
+
+Nostr event
+------------------
+This NIP specifies the use of the `1063` event type, having in `content` a description of the file content, and a list of tags described below:
+* `r` the url to download the file
+* `type` a string indicating the data type of the file. The MIME types format must be used (https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types)
+* `decrypt` (optional) If the file is encrypted, you must indicate in the first value the algorithm used to encrypt the file and in the second value the parameters that allow the decryption of the file
```

Comment:
```suggestion
* `xchacha20` (optional) If the file is encrypted using XChaCha20, you must indicate in the first value the encryption key and in the second value the nonce, as hex
```

Reply:
Are there decent libs for `xchacha20` ? void.cat uses AES-GCM

Reply:
I'm happy with AES-GCM. I just heard XChaCha20 is "better". I just think we need to decide on one.

Reply:
XChaCha20 is much newer so it's a bit harder to find decent libs, while AES ubiquitous